testing/src/test/java/org/motechproject/nms/testing/it/utils/SubscriptionHelper.java

```diff
@@ -67,6 +67,15 @@ public SubscriptionPack pregnancyPack() {
         return subscriptionService.getSubscriptionPack("pregnancyPack");
     }
 
+    public SubscriptionPack pregnancyPack(int messagesPerWeek) {
+        if (subscriptionPackDataService.byName("pregnancyPack") == null) {
+            createSubscriptionPack("pregnancyPack",
+                    SubscriptionPackType.PREGNANCY, PREGNANCY_PACK_WEEKS,
+                    messagesPerWeek);
+        }
+        return subscriptionService.getSubscriptionPack("pregnancyPack");
```

Comment:
What if the existing subscriptionPack does not have the same number of messages per week?